devito/function.py

```diff
@@ -135,7 +132,7 @@ def __init__(self, *args, **kwargs):
             super(TensorFunction, self).__init__(*args, **kwargs)
 
             # Staggered mask
-            self._staggered = kwargs.get('staggered', tuple(0 for _ in self.indices))
+            self._staggered = kwargs.get('staggered', tuple(None for _ in self.indices))
```

Comment:
I'm really struggling to understand why was this change necessary, which in turn requires a proliferation of
```
staggered = tuple([s if s is not None else 0 for s in self.staggered])
```
throughout the class, that I don't really like

Reply:
reminder